### ISSUE TITLE
Option to remove slow scroll / swipe switch pages functionality

### ISSUE THREAD

**Issue by AnataBakka** [tachiyomiorg/tachiyomi#6313]
### Describe your suggested feature

When you switch a page (left-right), as far as i can see, there's a different functionality whether you scroll (as in swipe while keeping your finger on the screen) fast or slow.
If you scroll slow, you only switch one page, and then get stuck.
If you scroll fast you can however indefinitely switch pages.

I hate when i try slowly switching pages and it doesn't, and i don't think it makes much sense having such a functionality exist.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by CrepeTF** [comment]
Could you send a video of this happening

**Comment by AnataBakka** [comment]
https://odysee.com/@AnataBakka:0/TachiyomiSwipingBehaviour:9
ok, actually, it seems like it was just my impression that you had to scroll fast to move pages.
maybe i'm just removing my finger when i do so.

i still think changing pages could go smoother though. no need for it to ever get stuck if you keep on scrolling

**Comment by arkon** [comment]
I honestly can't tell what you're doing in the video without actually seeing your hand/fingers.

Regardless though, this likely will never change unless it's fully rewritten. It's mostly just standard ViewPager behaviour.

**Comment by AnataBakka** [comment]
Can't you see the white circles when i press?
That's kinda the important part, you can see the circles (my fingers) move to the left, and yet the page doesn't scroll to the right, or viceversa.
It works kinda like when you zoom in. If you zoom in and then scroll to the right, your scroll will eventually get blocked by the side of the page, until you eventually remove your finger and start scrolling again to switch the page. Which is fine, as you don't want to accidentally switch page while reading one page. 

However i don't think it makes sense to do the same while switching pages. I feel like it's probably done for performance reasons, as it feels like you are only able to "see" 3 pages at a time, and once you reach the side of the last page, you have to remove your finger before you scroll again to do a removeFirst and addLast. You could probably be able to do the same with a loop check of "if reached the end of third page" while scrolling, but then that's probably expensive which is why the check is only added when you start scrolling, rather than while scrolling. Unless it was added "so you don't accidentally switch page while switching pages?"

I can understand if it can't be changed however.

**Comment by AnataBakka** [comment]
ps: @arkon is that a good reason to close this btw?
i think most of my issues i posted here are probably "standard ViewPager" behaviour. that still doesn't make it ok, and should be kept on track.
you say none of the reading functionality can be changed? that's like a very important thing, with the most issues (in my case)

**Comment by arkon** [comment]
> Can't you see the white circles when i press?

Yes, but that doesn't tell me how your hand(s?) are moving. It's not "normal" for somebody to try to scroll through pages with seemingly two fingers.

> is that a good reason to close this btw?

Because it realistically won't be changed so there's no point in tracking it. It'll be replaced in [the rewrite](https://github.com/tachiyomiorg/tachiyomi-1.x) anyway.
